backend/src/test/java/gov/cdc/usds/simplereport/db/model/FeatureFlagTest.java

```diff
@@ -0,0 +1,35 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class FeatureFlagTest {
```

Comment:
Can you remove the public modifier here?

Reply:
Similar to my comment above. If I set the feature flag field as private and I do not add the public modifier to the class some random not related unit test in the backend start failing.

Reply:
I dug into this a little more, and at this point I think it's a framework issue of some kind - we are using this `graphql-java-kickstart:graphql-spring-boot-starter` library to integrate graphQL into our setup, and this failure seems to be something of a known issue there: https://github.com/graphql-java-kickstart/graphql-spring-boot/discussions/419

I haven't been able to link it specifically to the use of `public`, but now that there's an official [SpringBoot/GraphQL integration](https://spring.io/projects/spring-graphql) I think we'd be better served by just migrating to that instead of trying to continue debugging here. Our test setup (`BaseGraphQLTest`) is also unlike any other test setup I've seen on the web...so that will be a fun puzzle to unravel once we start doing the migration.

So @johanna-skylight, if you could leave a comment on these files saying we should revisit `public` after the Spring for GraphQL migration is done, then I think we'll call this good for now. Thanks for your patience in investigating!

Reply:
added comments